File: CS/DevExpressMvcApplication1/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using DevExpress.XtraScheduler;

namespace DevExpressMvcApplication1.Views {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View(SchedulerDataHelper.DataObject);
        }

        public ActionResult SchedulerPartial() {
            ViewBag.ResourcesSortType = (Request.Params["ResourcesSortType"] != null) ? int.Parse(Request.Params["ResourcesSortType"]) : (int)ResourcesSortOrder.None;
            return PartialView("SchedulerPartial", SchedulerDataHelper.DataObject);
        }
    }
}

File: VB/DevExpressMvcApplication1/Controllers/HomeController.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports DevExpress.XtraScheduler

Namespace DevExpressMvcApplication1.Views
    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            Return View(SchedulerDataHelper.DataObject)
        End Function

        Public Function SchedulerPartial() As ActionResult
            ViewBag.ResourcesSortType = If(Request.Params("ResourcesSortType") IsNot Nothing, Integer.Parse(Request.Params("ResourcesSortType")), CInt(ResourcesSortOrder.None))
            Return PartialView("SchedulerPartial", SchedulerDataHelper.DataObject)
        End Function
    End Class
End Namespace
